CONTRIBUTING.md

```diff
@@ -85,6 +86,12 @@ Once you've set up your fork, you will find it useful to set up a git remote for
 
 Replace MYUSERNAME with your own GitHub username.
 
+## Choosing a version of python
+
+CVE Binary Tool supports [any version of python that has ongoing security support](https://endoflife.date/python).  There can be a bit of lag before we enable a new release or fully disable an old one.  If you don't need multiple versions of python for a specific bug, we recommend starting with either the most recent version of python or the one before (e.g. the first or second thing on [the "active python releases" list](https://www.python.org/downloads/)).
+
+Note that some of our [linters](#running-linters) (tools used to help with code quality) may not work on the oldest version of python.  That's the last one on [the "active python releases" list](https://www.python.org/downloads/).  So if you're doing development on the oldest version of python, you should switch to a newer version before you run linters on your code or do a `git commit` for best results.
```

Comment:
This last sentence is quite long so moving `for best results` to the beginning might make it easier to grasp on the first reading (or not, I'm not a native speaker myself!)

```suggestion
## Choosing a version of Python

CVE Binary Tool supports [any version of Python that has ongoing security support](https://endoflife.date/python).  There can be a bit of lag before we enable a new release or fully disable an old one.  If you don't need multiple versions of Python for a specific bug, we recommend starting with either the most recent version of Python or the one before (e.g. the first or second thing on [the "Active Python Releases" list](https://www.python.org/downloads/)).

Note that some of our [linters](#running-linters) (tools used to help with code quality) may not work on the oldest version of Python.  That's the last one on [the "Active Python Releases" list](https://www.python.org/downloads/).  So if you're doing development on the oldest version of Python, for best results you should switch to a newer version before you run linters on your code or do a `git commit`.
```